Script/server.js

```diff
@@ -25,16 +25,6 @@ function loadProjectsFromServer()
     })
 }
 
-function loadTicketsFromServer(id)
```

Comment:
I believe one of the 'if' uses loadTicketsFromServer() function, remove it

Reply:
Fixed